ncm-metaconfig/src/main/metaconfig/haproxy/backend.tt

```diff
@@ -0,0 +1,9 @@
+[% FOREACH option IN backend.value.options -%]
+option [% option %]
+[% END %]
+[% FOREACH check IN backend.value.tcpchecks -%]
+tcp-check [% CCM.unescape(check) %]
+[% END %]
+[% FOREACH server IN backend.value.servers -%]
+server [% server.hostname %] [% server.ip %] check port [% server.port %]
```

Comment:
The first value isn't necessarily a host-name, it's an arbitrary name for the back-end server, only the IP is used. It is particularly useful to support arbitrary names to distinguish between multiple services.

The check is optional, although it is a very good idea.

To quote the manual:
```
  server <name> <address>[:port] [settings ...]
```

Reply:
It would be really good to support the port after the server IP as well.

See http://cbonte.github.io/haproxy-dconv/1.8/configuration.html#4.2-server